student.rb

```diff
@@ -0,0 +1,12 @@
+require './main'
```

Comment:
- [OPTIONAL] It would be better to use `require_relative` instead of `require`. "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory." 😉

Reply:
@ArthurGC Thanks for the review and detailed suggestion. I have changed it to require_relative now.